### ISSUE TITLE
Not sure if "server-sent events" is the most appropriate wording?

### ISSUE THREAD

**Issue by staydecent** [balderdashy/sails-docs#585]
I may be wrong on this, but it gave me pause when reading [this page](http://sailsjs.org/documentation/reference/web-sockets/socket-client/io-socket-on) on `sails.io.js`:

> Starts listening for **server-sent events** from Sails with the specified eventIdentity. Will trigger the provided callback function when a matching event is received.

(emphasis is mine)

I believe *Server-sent events* (SSE) is a [different spec](https://html.spec.whatwg.org/multipage/comms.html#server-sent-events) that is not implemented in the `sails.io.js` library. This may cause confusion since (I think) the underlying tech is actually web sockets rather than SSE.

I realize the wording makes sense, it just, unfortunately, clashes with the name of a similar, yet different, spec. 

Not sure if this is inconsequential, but thought I'd share anyway.

**Comment by rachaelshaw** [comment]
@staydecent have any ideas for alternate wording?

**Comment by sailsbot** [comment]
Thanks for posting, @staydecent.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
